README.md

```diff
@@ -64,6 +64,11 @@ file (assuming the `erlang_ls` executable is available in your `PATH`:
     ;; Enable LSP automatically for Erlang files
     (add-hook 'erlang-mode-hook #'lsp)
 
+    ;; TCP connection timeout
+    (setq lsp-tcp-connection-timeout 500)
```

Comment:
Can we not set this timeout in `lsp-erlang`?

Reply:
We could, although it's not really necessary to do so. 

I removed this option form the  document in a later PR [here](https://github.com/erlang-ls/erlang_ls/pull/273), since this is more specific to what the Emacs user wants as a timeout in general and not specific for `erlang-ls` to work.